test/metabase/query_processor/expand_resolve_test.clj

```diff
@@ -19,6 +23,36 @@
                                {k (obj->map v)}))
     :else           o))
 
+(def ^:private resolve'
```

Comment:
Even by reading the code for this function I'm not really sure what's it's supposed to. It's not really clear from the name (`resolve'`) so it would be helpful to me or anyone else who has to work on it in the future to have either a more descriptive name or a short docstring or even just a comment explaining the purpose